app/src/main/java/vn/asiantech/com/android_circleci_sample/MainActivity.java

```diff
@@ -5,6 +5,8 @@
 
 public class MainActivity extends AppCompatActivity {
 
+    public int A;
```

Comment:
WARNING: [SuspiciousConstantFieldName] The field name indicates a constant but its modifiers do not
http://pmd.sourceforge.net/pmd-5.2.3/pmd-java/rules/java/naming.html#SuspiciousConstantFieldName